tools/representative_length.py

```diff
@@ -3,7 +3,7 @@
 from shapely.geometry import MultiLineString, LineString
 from shapely.geometry import MultiPoint
 from shapely.ops import split
-
+import arcpy
```

Comment:
het idee is dat we deze gistools kunnen runnen zonder arcGIS. Beter is dus om een string met opmerkingen te verzamelen en deze terug te geven. En deze vervolgens terug te geven, waarna ze in TijhuisArcGisTools  met arcpy geprint kunnen worden

Reply:
Dit is denk ik vanuit een andere testcase. Ik zie de arcpy nu niet meer in de code staan.